src/overlay.jsx

```diff
@@ -76,12 +76,14 @@ const Overlay = React.createClass({
   },
 
   getStyles() {
+    const rawTheme = this.state.muiTheme.rawTheme;
+
     let styles = {
       root: {
         position: 'fixed',
         height: '100%',
         width: '100%',
-        zIndex: 9,
+        zIndex: rawTheme.zIndex.dialog - 1,
```

Comment:
I think it's better to have `dialogOverlay` and `leftNavOverlay` too. dialog - 1 = leftnav. that means it's possible that dialog's overlay might not cover leftnav's in some cases, which is undesirable